docs/explanation/security.md

```diff
@@ -0,0 +1,24 @@
+# Security
+
+
+## Access to the API
```

Comment:
Just to be sure: Do we want to move this section to the beginning of the API spec, or at least have a link there pointing to this?

Reply:
My idea is that we'll keep the "Access to the API" section in the new Security doc, as an entry-point for people looking for security info. We should be able to replace [API and clients](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/api-and-clients/) by content in the API reference, so when the API reference is ready we can change the link at the end of "Access to the API" to point to the API reference instead of [API and clients](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/api-and-clients/).

Does it seem reasonable?